src/main/java/net/openhft/chronicle/queue/impl/single/PretoucherState.java

```diff
@@ -104,16 +104,23 @@ public void pretouch(MappedBytes bytes) {
                         // ignored.
                     }
                     try {
+                        if (bytes != null && lastTouchedPage >= realCapacity) {
```

Comment:
I suggest not doing any of these changes in Chronicle-Queue repository, rather moving these classes as is to CQE and then changing the classes there. What do you think?

Reply:
This has now been replaced by another PR